Documentation/Index.rst

```diff
@@ -1,69 +1,55 @@
+..  include:: /Includes.rst.txt
 
-=============
-render-guides
-=============
+.. _start:
 
-Mono-repo setup
-===============
+=========
+<project>
```

Comment:
We should substitute this placeholder, although it is a duplicate of the package name.

```suggestion
render-guides
```